warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE"

Compiled on Debian
```
In file included from /usr/include/fcntl.h:25:0,                                                                                                        
                 from /root/co2mon/co2mond/src/main.c:23:                                                                                               
/usr/include/features.h:148:3: warning: #warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE" [-Wcpp]                             
 # warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE"                                                                           
   ^~~~~~~                                                                                                                          
```

--- a/co2mond/src/main.c
+++ b/co2mond/src/main.c
@@ -16,8 +16,9 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-#define _BSD_SOURCE
-#define _XOPEN_SOURCE 700
+#define _XOPEN_SOURCE 700 /* strnlen */
+#define _BSD_SOURCE /* daemon() in glibc before 2.19 */
+#define _DEFAULT_SOURCE /* _BSD_SOURCE is deprecated in glibc 2.19+ */
 #define _DARWIN_C_SOURCE /* daemon() on macOS */
 
 #include <fcntl.h>

